json-rpc/types/src/proto/jsonrpc.proto

```diff
@@ -285,10 +285,15 @@ message Script {
    * - List<u8> value => "{U8Vector:: 0x<hex-encoded bytes>}"
    */
   repeated string arguments = 3;
+  /**
+   * List of hex-encoded string of BCS bytes representing script function arguments.
+   * This field does not contain type information.
+   */
+  repeated string arguments_bcs = 4;
   /**
    * List of type arguments, converted into string.
    */
-  repeated string type_arguments = 4 [json_name="type_arguments"];
+  repeated string type_arguments = 5 [json_name="type_arguments"];
```

Comment:
you can only append new field, do not update existing one.
so what you should do here is add repeated string arguments_bcs = 13 at the end of this struct.

Reply:
you also need add [json_name="arguments_bcs"]

Reply:
Fixed in d70bece